integration-cli/docker_api_test.go

```diff
@@ -15,24 +15,6 @@ import (
 	"gotest.tools/assert"
 )
 
-func (s *DockerSuite) TestAPIOptionsRoute(c *check.C) {
-	resp, _, err := request.Do("/", request.Method(http.MethodOptions))
-	assert.NilError(c, err)
-	assert.Equal(c, resp.StatusCode, http.StatusOK)
-}
-
-func (s *DockerSuite) TestAPIGetEnabledCORS(c *check.C) {
-	res, body, err := request.Get("/version")
-	assert.NilError(c, err)
-	assert.Equal(c, res.StatusCode, http.StatusOK)
-	body.Close()
-	// TODO: @runcom incomplete tests, why old integration tests had this headers
-	// and here none of the headers below are in the response?
-	//c.Log(res.Header)
-	//assert.Equal(c, res.Header.Get("Access-Control-Allow-Origin"), "*")
```

Comment:
Looks like this behaviour changed in https://github.com/moby/moby/pull/32174, where the middleware was only added if a non-empty value is set for the CORS headers.

Interestingly though, the middleware has a special case for empty strings, and changes that to use `"*"`; https://github.com/moby/moby/blob/8e610b2b55bfd1bfa9436ab110d311f5e8a74dcb/api/server/middleware/cors.go#L24-L35

So I'm wondering if 

- the `if cfg.CorsHeaders != ""` should be removed 🤔 (OTOH, `*` and "not set" could be considered equivalent?)
- otherwise; if the special case (change `""` to `"*"`) should be removed